tests/cases/data/source/MongoDbTest.php

```diff
@@ -272,166 +448,6 @@ public function testConditionsWithSchema() {
 		$this->assertEqual($expected, $result);
 	}
 
-	public function testReadNoConditions() {
```

Comment:
The crud operations are tested in the `Crud` base test, so as along as we don't check for any mongo specialities, they are good to be removed here.